src/test/java/com/example/desafio_quality/service/PropertyServiceTest.java

```diff
@@ -108,4 +125,40 @@ public void getBiggestRoom_returnsExceptionNotFound_withNoRooms(){
             service.getBiggestRoom(propertyWithNoRooms.getPropId());
         });
     }
+
+    @Test
+    @DisplayName("Verifica se retorna o valor total do imóvel")
+    void getValue_returnCorrectValue() {
+        Mockito.when(repo.getPropertyById(ArgumentMatchers.anyInt()))
+                .thenReturn(Optional.ofNullable(property));
+        Mockito.when(repoDistrict.getDistrictById(ArgumentMatchers.anyInt()))
+                .thenReturn(Optional.ofNullable(district));
+
+        PropertyValueDTO propertyValue = service.getValue(1);
+
+        assertThat(propertyValue).isNotNull();
+        assertThat(propertyValue.getValue()).isEqualTo(new BigDecimal("17280.0000"));
+    }
+
+    @Test
+    @DisplayName("Valida se retorna um erro caso o id da propriedade seja inválido")
+    void getValue_returnsNotFoundException_withIncorrectPropertyId() {
+        Mockito.when(repo.getPropertyById(ArgumentMatchers.anyInt()))
+                .thenReturn(Optional.empty());
+        assertThrows(NotFoundException.class, () -> {
+            service.getValue(1);
```

Comment:
Esse 1 do parâmetro é referente ao id da propriedade, certo? Se sim, um padrão que estamos utilizando, é pegar esse id do objeto propriety, fazendo algo semelhante a: `this.propriety.getPropId()`.

Isso evita Magic Number